### DISCUSSION TITLE
Silicon support

### DISCUSSION THREAD

**Discussion by philipmclifton** [spatie/ray#390] [Ideas]
Long shot but are there any plans to add support for M1 processor?

**Comment by freekmurze** [comment]
Yup, we'll do so in the coming weeks 👍

**Comment by freekmurze** [comment]
M1 support has arrived: https://twitter.com/freekmurze/status/1381710165808975892

**Reply by philipmclifton** [reply]
Awesome thanks! Loaded up with m1 speed, il let you know if I find any issues. 😄
